rsocket-core/src/main/java/io/rsocket/fragmentation/FragmentationUtils.java

```diff
@@ -0,0 +1,34 @@
+package io.rsocket.fragmentation;
+
+import io.netty.buffer.ByteBuf;
+import io.rsocket.Payload;
+import io.rsocket.frame.FrameHeaderFlyweight;
+import io.rsocket.frame.FrameLengthFlyweight;
+
+public final class FragmentationUtils {
```

Comment:
This class is for an internal check. It would be better to avoid a whole new public class. An alternative is to create a `BiPredicate<int, Payload> payloadLengthPredicate` in `RSocketFactory` and pass it into requester and responder instead of `int mtu`.

Reply:
Instead, moved validator utils to core package and made it pkg private